test/schemas/ecommerceSchemas.test.ts

```diff
@@ -136,8 +136,7 @@ describe('The product details schema', () => {
             productDetails.validate(value);
         }
 
-        expect(validate).toThrow(Error);
-        expect(validate).toThrow(message);
+        expect(validate).toThrow(new Error(message));
```

Comment:
These changes from two checks to one check are not the same thing. Testing separately ensures the thrown object is an actual `Error` instance while having it inline doesn't:

https://replit.com/@Fryuni/jest-error-exposition

`jest-extended` has a test that is actually equivalent:
```ts
expect(validate).toThrowWithMessage(Error, message);
```